pkg/fileutils/fileutils_linux.go

```diff
@@ -0,0 +1,18 @@
+package fileutils
+
+import (
+	"os"
+
+	"github.com/docker/docker/pkg/system"
+)
+
+// CurrentUserIsOwner checks whether the current user is the owner of the given
+// file.
+func CurrentUserIsOwner(f string) (bool, error) {
```

Comment:
I agree with you that's it's kinda weird to have a function returning only true, nil(error) and another false, error though